AccountService.js

```diff
@@ -23,8 +32,16 @@ export class AccountService {
         }
       });
       return true;
-    } catch(e) {}
-    return false;
+    } catch(e) {
+      if(!e.response || !e.response.status) {
```

Comment:
if `e.response` is not defined, you're going throw "can't access status of undefined error" here.

Reply:
my apologies corrected it.